docs/product/releases/setup/release-automation/circleci/index.mdx

```diff
@@ -61,7 +61,7 @@ notify-sentry-deploy:
           sentry-cli releases set-commits $SENTRY_RELEASE --auto
           sentry-cli releases files $SENTRY_RELEASE upload-sourcemaps path-to-sourcemaps-if-applicable
           sentry-cli releases finalize $SENTRY_RELEASE
-          sentry-cli releases deploys $SENTRY_RELEASE new -e $SENTRY_ENVIRONMENT
+          sentry-cli deploys new -e $SENTRY_ENVIRONMENT
```

Comment:
The `deploys` commands automatically use the `$SENTRY_RELEASE` environment variable, hence why the `--release` flag can be omitted here.